front/components/poke/subscriptions/table.tsx

```diff
@@ -31,11 +49,190 @@ export function SubscriptionsDataTable({
 }: SubscriptionsDataTableProps) {
   return (
     <div className="border-material-200 my-4 flex flex-col rounded-lg border p-4">
-      <h2 className="text-md mb-4 font-bold">Subscriptions:</h2>
+      <h2 className="text-md mb-4 font-bold">History of subscriptions:</h2>
       <PokeDataTable
         columns={makeColumnsForSubscriptions()}
         data={prepareSubscriptionsForDisplay(owner, subscriptions)}
       />
     </div>
   );
 }
+
+export function ActiveSubscriptionTable({
+  subscription,
+}: {
+  subscription: SubscriptionType;
+}) {
+  const activePlan = subscription.plan;
+  return (
+    <div className="flex flex-col space-y-8 pt-4">
```

Comment:
Any chance we could factorize this? Like having an object like we do for `data_table`?

Reply:
Yes! Merging as is so I can focus on the new Enterprise form this morning, I'll open a new PR to handle this feedback right after 🙇🏻‍♀️